### PR TITLE
Add section about replace with functionality

### PR DESCRIPTION
**Pull request by ripla**
This change is [Reviewable](https://reviewable.io/reviews/vaadin/designer-documentation/7)

### CODE REVIEW

**Comment by anssit** [comment]
Some observations:
- title should be smaller (same size as replace with)
- title could read just "replacing a layout". The "with another layout" part could go into text if you want to mention it explicitly.
- screenshot should be smaller

---

Review status: 0 of 2 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/designer-documentation/7#-:-Kgxl1XioVtFfuKS2FZI:b8x46q)*

**Comment by anssit** [comment]
Review status: 0 of 2 files reviewed at latest revision, 3 unresolved discussions.

---

*[designer-designing.asciidoc, line 207 at r2](https://reviewable.io:443/reviews/vaadin/designer-documentation/7#-Kgxnf6FwM-dd0V37fTP:-Kgxnf6FwM-dd0V37fTQ:b-rsglxw) ([raw file](https://github.com/vaadin/designer-documentation/blob/cd79e3de42cd10460bae5d71b376fadefa3c7e83/designer-designing.asciidoc#L207)):*
> ```AsciiDoc
> 
> In many cases a change in orientation or other rearrangement of components is
> required. Replacing the layout that contains those components is the best
> ```

Instead of "best" - maybe "a convenient way to achieve that" or something else a bit more modest. 

---

*[designer-designing.asciidoc, line 209 at r2](https://reviewable.io:443/reviews/vaadin/designer-documentation/7#-KgxmvqPvNHkZZSWI5H6:-KgxmvqPvNHkZZSWI5H7:bsr7cz3) ([raw file](https://github.com/vaadin/designer-documentation/blob/cd79e3de42cd10460bae5d71b376fadefa3c7e83/designer-designing.asciidoc#L209)):*
> ```AsciiDoc
> required. Replacing the layout that contains those components is the best
> option. To replace a layout, click on the layout to be replaced in the
> [guilabel]#Outline# view, choose [menuchoice]#Replace with# in the context menu 
> ```

Maybe add the three dots as well: "Replace with..."

---

*[designer-designing.asciidoc, line 219 at r2](https://reviewable.io:443/reviews/vaadin/designer-documentation/7#-Kgxn0r_sjDIn_WBO7f6:-Kgxn0r_sjDIn_WBO7f7:bwevnj1) ([raw file](https://github.com/vaadin/designer-documentation/blob/cd79e3de42cd10460bae5d71b376fadefa3c7e83/designer-designing.asciidoc#L219)):*
> ```AsciiDoc
> 
> Note that usual layout restrictions apply. For example if you replace a 
> [classname]#ComponentContainer# with a [classname]#SingleComponentContainer# 
> ```

I don't see SingleComponentContainer in the list. What's that? (Ok, I know, but maybe not all Designer users know.) Maybe rephrase in terms of available layouts. Like: If you replace a VerticalLayout that contains three Buttons with a Panel, only the first Button in the VerticalLayout is moved to Panel and the rest are dropped as Panel only accepts one child component. Well, that was a bit long.. but something along those lines.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/designer-documentation/7)*

**Comment by ripla** [comment]
Review status: 0 of 2 files reviewed at latest revision, 3 unresolved discussions.

---

*[designer-designing.asciidoc, line 207 at r2](https://reviewable.io:443/reviews/vaadin/designer-documentation/7#-Kgxnf6FwM-dd0V37fTP:-Kgxod8sBPOaAgfvhPb0:b-896fix) ([raw file](https://github.com/vaadin/designer-documentation/blob/cd79e3de42cd10460bae5d71b376fadefa3c7e83/designer-designing.asciidoc#L207)):*
<details><summary><i>Previously, anssit wrote…</i></summary><blockquote>

Instead of "best" - maybe "a convenient way to achieve that" or something else a bit more modest. 
</blockquote></details>

Done.

---

*[designer-designing.asciidoc, line 209 at r2](https://reviewable.io:443/reviews/vaadin/designer-documentation/7#-KgxmvqPvNHkZZSWI5H6:-KgxogEEViFF1Z7FufsI:b-896fix) ([raw file](https://github.com/vaadin/designer-documentation/blob/cd79e3de42cd10460bae5d71b376fadefa3c7e83/designer-designing.asciidoc#L209)):*
<details><summary><i>Previously, anssit wrote…</i></summary><blockquote>

Maybe add the three dots as well: "Replace with..."
</blockquote></details>

Done.

---

*[designer-designing.asciidoc, line 219 at r2](https://reviewable.io:443/reviews/vaadin/designer-documentation/7#-Kgxn0r_sjDIn_WBO7f6:-KgxpzK8YQ8MD44Cqykf:b-896fix) ([raw file](https://github.com/vaadin/designer-documentation/blob/cd79e3de42cd10460bae5d71b376fadefa3c7e83/designer-designing.asciidoc#L219)):*
<details><summary><i>Previously, anssit wrote…</i></summary><blockquote>

I don't see SingleComponentContainer in the list. What's that? (Ok, I know, but maybe not all Designer users know.) Maybe rephrase in terms of available layouts. Like: If you replace a VerticalLayout that contains three Buttons with a Panel, only the first Button in the VerticalLayout is moved to Panel and the rest are dropped as Panel only accepts one child component. Well, that was a bit long.. but something along those lines.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/designer-documentation/7)*

**Comment by anssit** [comment]
:lgtm: 

---

Reviewed 1 of 2 files at r1, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/designer-documentation/7#-:-KgxqtDXvJFMLUaIHdIB:bg7igkv)*
